torch/_utils.py

```diff
@@ -592,6 +592,25 @@ def dec(fun):
     return dec
 
 
+def render_call(fn, args, kwargs):
+    import torch._tensor_str
+    from torch.overrides import resolve_name
```

Comment:
Is this torch/__init__.py import order issue? Why not just use `torch.overrides.resolve_name` directly below?

Reply:
Mostly (1) I didn't want to assume that overrides was imported automatically from torch (it is, in fact) and (2) since this is a utils module I wanted to be sensitive to import order issues (but I don't think there are any)